kratos/modeler/operation/voxel_mesher_operation.cpp

```diff
@@ -54,12 +54,26 @@ ModelPart& VoxelMesherOperation::CreateAndGetModelPart(std::string const& rFullN
     return mrModeler.CreateAndGetModelPart(rFullName);
 }
 
+std::size_t VoxelMesherOperation::GetNodeIndex(std::size_t I, std::size_t J, std::size_t K) const
```

Comment:
```suggestion
std::size_t VoxelMesherOperation::GetNodeIndex(
    std::size_t I,
    std::size_t J,
    std::size_t K) const
```
Minor style comment.